support-frontend/assets/helpers/utilities/dateFormatting.js

```diff
@@ -0,0 +1,28 @@
+// @flow
```

Comment:
These helpers are copied from `lpPreviousGiving.js`, as they're now being used in two places